ui/server/src/test/scala/pl/touk/nussknacker/ui/integration/BaseFlowTest.scala

```diff
@@ -214,6 +217,56 @@ class BaseFlowTest extends FunSuite with ScalatestRouteTest with FailFastCirceSu
     }
   }
 
+  test("should reload ConfigCreator") {
```

Comment:
what about test for process with reloaded definition? would be nice to show in test that invoking `reload` in api doesn't change definition of proces - you still need to redeploy it...

Reply:
Do you mean definition of process (json in DB)? Or process running in Flink?

Reply:
Let's say that in this reload change type returned by source - you can use now use brand new field "foo". I would like to see in tests that after reload I can introduce change in process which will use this new field and after redeploy it will handle it. Isn't it a purpose of this change? Definition reload is not the intention of user - he/she want to use this definition. I know that it works almost for sure but maybe we have some caching of part or definition somewhere or some problems with serialization. Without test it is hard to say...

Reply:
You're right :+1: I added more elaborate test - is this what you had in mind?